input/fsh/Terminology.fsh

```diff
@@ -1,34 +1,36 @@
-// DEMOGRAPHICS CodeSystems
-
 Alias: $FHIRMS = http://terminology.hl7.org/CodeSystem/v3-MaritalStatus
+Alias: $SCT = http://snomed.info/sct // standard set is published separately for re-use in other IGs
+
+///////////////////////////////////////////////////// DEMOGRAPHICS CodeSystems \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
 
-ValueSet: DemographicFactorsSex
-Id: DemographicFactorsSex
-Title: "Sex of patient"
-Description: "Codes used in sex of patient"
+ValueSet: DemographicFactorsSexAtBirth
+Id: DemographicFactorsSexAtBirth
+Title: "The patient's sex at birth"
+Description: "Valueset of the patient's sex at birth"
 * include codes from system http://hl7.org/fhir/administrative-gender
 
 ValueSet: DemographicFactorsCountry
 Id: DemographicFactorsCountry
-Title: "Country"
-Description: "Country (e.g. of residence)"
+Title: "Country of residence of patient"
+Description: "Valueset of the country (e.g. of residence) of a patient"
 * include codes from valueset http://hl7.org/fhir/ValueSet/iso3166-1-3
 
 ValueSet: DemographicFactorsEthnicity
 Id: DemographicFactorsEthnicity
-Title: "Ethnicity"
-Description: "Codes used in ethnicity of patient"
-* include codes from system http://terminology.hl7.org/CodeSystem/v3-Ethnicity
+Title: "Ethnicity of patient"
+Description: "Valueset of the ethnicity of a patient"
+* include codes from valueset http://terminology.hl7.org/ValueSet/v3-Ethnicity
+// this valueset is not rendering in the SDC form 
```

Comment:
I've had a look but it seems to be an LHCForms bug. Reported it @ https://github.com/lhncbc/lforms-fhir-app/issues/22 and [tracking internally](https://ichom.atlassian.net/browse/BCFI-59).